ma_dsn.c

```diff
@@ -86,6 +86,7 @@ MADB_DsnKey DsnKeys[]=
   {"APP",            offsetof(MADB_Dsn, App),               DSN_TYPE_STRING, 0, 0},
   /* SSO parameters */
   {"BROWSER_SSO",    offsetof(MADB_Dsn, IsBrowserAuth),     DSN_TYPE_BOOL  , 0, 0},
+  {"NO_KEYRING_SSO", offsetof(MADB_Dsn, HasNoKeyring),      DSN_TYPE_BOOL  , 0, 0},
```

Comment:
nit:
What about `IGNORE_KEYRING_SSO` and `IgnoreKeyring` or smth like this?

Reply:
fine with me